### ISSUE TITLE
TynyMCE editor not disappearing

### ISSUE THREAD

**Issue by shawnboy777** [voidlabs/mosaico#374]
Thanks for all of the hard work that went into this system  It is absolutely amazing!!

I have found that when editing text elements, the TinyMCE editor is not being fully un-initiated when you click out of the editing area.  It does disappear when you first click out of the box, but then when you scroll, the editor appears again and seems to follow where you had last edited.  I tried this in Firefox and Safari, as well as with nodejs and PHP  backends.  What can be done to fix this?

**Comment by shawnboy777** [comment]
I figured out how to make the mysteriously reappearing TinyMCE editor not do that when you scroll.  In the /src/js/bindings/scrollfix.js file, I commented out the line tinymce.activeEditor.theme.panel.visible(true); (near line 29) because when the element moves via "nodeChanged" it would automatically make the visibility "true."

I don't know if this will cause issues in the future, but it fixed my problem now.

**Comment by bago** [comment]
I can't confirm the behaviour that you describe.

Does it happen on "mosaico.io" demo site?
Which version of tinymce are you using?

The fix in "scrollfix" is there for a reason: without that code sometimes the toolbar disappear and never appears again. So, if you need that "fix" we should better understand if others can reproduce your issue.

**Comment by shawnboy777** [comment]
Thanks for looking into this for me.  I certainly was not aware of the scrollfix issue, since the bar always appeared for me after I commented out that line. 

I have successfully reproduced the issue on https://mosaico.io using the versafix-1 template.  It occurs on Mac and Windows, using any browser (Firefox, Chrome, Safari, Internet Explorer).  Here is a screen capture of the issue:
https://www.dropbox.com/s/fzsugz3iw6n2d7p/Mosaico_TinyMCE_toolbar.mov?dl=0 

I think that the scrollfix "solution" could be conditional (if the toolbar is not visible and the nodeChange happens then keep the toolbar invisible), but I was not able to truly figure out how to do that.
